README.md

```diff
@@ -36,7 +36,7 @@
 
 ## Requirements
 
-* PHP 7.0+
+* PHP 7.2+
```

Comment:
Curious on why we need to bump to 7.2?

Reply:
The lib ([`ua-parser/uap-php`](https://github.com/ua-parser/uap-php)) we are using for UA parser requires PHP >= 7.2